dev/admissions-page/admissions.js

```diff
@@ -25,10 +25,17 @@ var enableScrollSpy = function($) {
   $('.animate-scroll').click(function(e) {
     e.preventDefault();
 
+    var $sectionsMenu = $('#sections-menu');
     var $target = $(this.hash);
     $target = $target.length ? $target : $('[name=' + this.hash.slice() + ']');
 
-    var scrollTo = $target.offset().top - 50;
+    var scrollTo = $target.offset().top;
+
+    if ($sectionsMenu.length > 0) {
+      scrollTo = scrollTo - $sectionsMenu.height() + 20;
```

Comment:
Adding some sort of grouping may be helpful here (e.g. `scrollTo - ($sectionsMenu.height() + 20)` or `(scrollTo - $sectionsMenu.height()) + 20`), depending on what you're trying to accomplish.